charts/vela-minimal/templates/defwithtemplate/config-helm-repository.yaml

```diff
@@ -0,0 +1,59 @@
+# Code generated by KubeVela templates. DO NOT EDIT. Please edit the original cue file.
+# Definition source cue file: vela-templates/definitions/internal/config-helm-repository.cue
+apiVersion: core.oam.dev/v1beta1
+kind: ComponentDefinition
+metadata:
+  annotations:
+    custom.definition.oam.dev/name.config.oam.dev: Helm Repository
+    definition.oam.dev/description: Config information to authenticate helm chart repository
+  labels:
+    custom.definition.oam.dev/catalog.config.oam.dev: velacore-config
+    custom.definition.oam.dev/multi-cluster.config.oam.dev: "true"
+    custom.definition.oam.dev/type.config.oam.dev: helm-repository
+  name: config-helm-repository
+  namespace: {{ include "systemDefinitionNamespace" . }}
+spec:
+  schematic:
+    cue:
+      template: |
+        output: {
+        	apiVersion: "v1"
+        	kind:       "Secret"
+        	metadata: {
+        		name:      parameter.name
+        		namespace: "vela-system"
+        		labels: {
+        			"config.oam.dev/catalog":       "velacore-config"
+        			"config.oam.dev/type":          "helm-repository"
+        			"config.oam.dev/multi-cluster": "true"
+        			"config.oam.dev/identifier":    parameter.registry
+        		}
+        	}
+        	type: "Opaque"
+
+        	if parameter.https != _|_ {
+        		stringData: parameter.https
+        	}
+        	if parameter.ssh != _|_ {
+        		stringData: parameter.ssh
+        	}
+        }
+        parameter: {
+        	oss?: {
+        		bucket:   string
+        		endpoint: string
+        	}
+        	https?: {
+        		url:      string
+        		username: string
+        		password: string
```

Comment:
username and password are optional.

Reply:
Do you mean some chart repository is not authorized,like https://github.com/oam-dev/kubevela/pull/3430/files/a3ec059aa8a2e1e44c5a1a39977549d270d88484#diff-be37e1f045977f8357845bf7beb8cefe87b937331d803f7c2457680d4763c319R61?

Reply:
Yse, in fact most chart repo is public. Everyone can access without any secret